### PR TITLE
Self PeerId can no longer be added

### LINKED ISSUE #48: do not allow to add ur own peer in convo
Disallow adding our own peer id as a new convo

### PR DESCRIPTION
**Pull request by jjperezaguinaga**
Also included [commitzen](https://www.npmjs.com/package/commitizen) to automate our commit messages and generate changelogs automatically.

Fix #48

### CODE REVIEW

**Comment by tolbrino** [comment]
Don't forget the PR metadata. And how is cz meant to be used/work?

**Review by nionis — APPROVED** [review]

**Comment by jjperezaguinaga** [comment]
> Don't forget the PR metadata. And how is cz meant to be used/work?

The PR is connected to an issue, so not sure what might be missing. `cz` was meant to be added as a way to streamline the way we add commits. Should have been a separate commit tbh but I think we can still sneak it as part of this minor fix.

File: src/components/new-conversation.tsx [hunk 1]
```diff
@@ -1,15 +1,44 @@
...
+        undefined
+
+  const REJECTIONS = {
+    'ALREADY_EXISTS': 'PeerID already exists',
+    'SAME_PEER_ID': 'You can not add your own PeerId'
+  }
+
+  const hasRejection = Boolean(rejection)
+
+  const rejectedMessage = (msg: string) => (
```
[the remark this answers is not in the document]
Comment (jjperezaguinaga) [reply src/components/new-conversation.tsx:37]: Sure, but why would it be? It's not used anywhere else.

[the remark this answers is not in the document]
Comment (nionis) [reply src/components/new-conversation.tsx:37]: it's very minor and has no real performance impact, but to basically not reinitialize the function every re-render

**Pull request merged**